src/app/api/user/add-user/route.ts

```diff
@@ -0,0 +1,23 @@
+import { sql } from "@vercel/postgres"
+import { NextResponse } from "next/server"
+import { encryptPassword } from "@/app/modules/encrpytion"
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url)
+
+  const firstName = searchParams.get("firstName")
+  const lastName = searchParams.get("lastName")
+  const email = searchParams.get("email")
+  const hashed_password = encryptPassword(searchParams.get("password"))
+
+  try {
+    if (!email || !hashed_password)
+      throw new Error("email and password required")
+    await sql`INSERT INTO users (first_name, last_name, email, hashed_password) VALUES (${firstName}, ${lastName}, ${email} ,${hashed_password});`
```

Comment:
So normally we NEVER want to embed raw text from a user input into a sql query for fear of sql injections. That said, the sql template literal provided by vercel does query sanitation for us. 

If we didn't have vercel we'd want to abstract our variables in such a way as to prevent someone from hijacking our database with malicious queries.